### PR TITLE
Get winner test

### PR DESCRIPTION
**Pull request by arnaldoaparicio**
## Checklist
- [x] Tests written where required
- [x] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  Test for get winner has been written.
What (if any) features are you implementing?

What (if anything) did you refactor?

Were there any issues that arose?

Is there anything that you need from your teammate?

Any other comments, questions, or concerns?

### CODE REVIEW

File: spec/graphql/queries/submissions/get_winner_spec.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    it 'returns a winner' do
+      admin = create(:admin, id: 1)
+      create_list(:submission, 5)
+      first_submission = Submission.all.first
+
+      response = TroubadourBeSchema.execute(query)
+      parsed = json_parse(response.to_json)
+
+      expect(parsed[:data][:getWinner]).to eq(nil)
```
Comment (tjroeder) [inline spec/graphql/queries/submissions/get_winner_spec.rb:39]: Nice test before you change the winner!

File: spec/graphql/queries/submissions/get_winner_spec.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      first_submission.update(winner: true)
+
+      new_response = TroubadourBeSchema.execute(query)
+      new_parsed = json_parse(new_response.to_json)
+
+      expect(new_parsed[:data][:getWinner][:winner]).to eq(true)
+      expect(new_parsed[:data][:getWinner][:id].to_i).to eq(first_submission.id)
+    end
+
+    it 'returns a different winner' do
```
Comment (tjroeder) [inline spec/graphql/queries/submissions/get_winner_spec.rb:49]: Also, testing changing the winner for our MVP design! Great work.

**Review by tjroeder — APPROVED** [review]
Great tests. Nice work.

**Review by Sierra-T-9598 — COMMENTED** [review]
Awesome, looks amazing!!

**Pull request merged**